app/components/Form/ApplicationForm.tsx

```diff
@@ -157,6 +158,9 @@ const ApplicationForm: React.FC<Props> = ({
     query
   );
   const { id, rowId, formData, status, updatedAt } = application;
+  
+  Settings.defaultZone = "America/Vancouver";
+  Settings.defaultLocale = "en_CA";
```

Comment:
Can these lines be removed now that we're setting in in `_app`?

Reply:
Removed